test/functional/feature_block.py

```diff
@@ -149,60 +134,60 @@ def get_tests(self):
         self.move_tip(1)
         b3 = self.next_block(3, spend=out[1])
         txout_b3 = PreviousSpendableOutput(b3.vtx[1], 0)
-        yield self.rejected()
+        self.sync_blocks([b3], False)
 
         # Now we add another block to make the alternative chain longer.
         #
         #     genesis -> b1 (0) -> b2 (1)
         #                      \-> b3 (1) -> b4 (2)
         self.log.info("Reorg to a longer chain")
-        self.next_block(4, spend=out[2])
-        yield self.accepted()
+        b4 = self.next_block(4, spend=out[2])
+        self.sync_blocks([b4])
 
         # ... and back to the first chain.
         #     genesis -> b1 (0) -> b2 (1) -> b5 (2) -> b6 (3)
         #                      \-> b3 (1) -> b4 (2)
         self.move_tip(2)
-        self.next_block(5, spend=out[2])
+        b5 = self.next_block(5, spend=out[2])
         self.save_spendable_output()
-        yield self.rejected()
+        self.sync_blocks([b5], False)
 
         self.log.info("Reorg back to the original chain")
-        self.next_block(6, spend=out[3])
-        yield self.accepted()
+        b6 = self.next_block(6, spend=out[3])
+        self.sync_blocks([b6], True)
 
         # Try to create a fork that double-spends
         #     genesis -> b1 (0) -> b2 (1) -> b5 (2) -> b6 (3)
-        #                                          \-> b7 (2) -> b8 (4)
+        #                                          \-> b7 (3) -> b8 (4)
```

Comment:
Why are you changing this comment?

Block 7 spends the output 2 (c.f. line 165)

Reply:
Yes - you're right. Fixed